From 45e31ba2c9c7f8d00f4ab8837ec54c4d7ebc809b Mon Sep 17 00:00:00 2001
From: Valentine Barshak <valentine.barshak@cogentembedded.com>
Date: Sat, 29 Jun 2019 16:49:34 +0300
Subject: [PATCH 163/211] arm64: dts: renesas: r8a77970-v3msk: Add ethernet0
 alias

This adds ethernet0 alias to the avb. This is needed
for U-Boot to pass the MAC address via the device tree.
The MAC address registers are cleared when the device
is disabled in U-Boot before starting the kernel.

Signed-off-by: Valentine Barshak <valentine.barshak@cogentembedded.com>
---
 arch/arm64/boot/dts/renesas/r8a77970-v3msk.dts | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/renesas/r8a77970-v3msk.dts b/arch/arm64/boot/dts/renesas/r8a77970-v3msk.dts
index af6c7b9..adc2b01 100644
--- a/arch/arm64/boot/dts/renesas/r8a77970-v3msk.dts
+++ b/arch/arm64/boot/dts/renesas/r8a77970-v3msk.dts
@@ -15,6 +15,7 @@
 
 	aliases {
 		serial0 = &scif0;
+		ethernet0 = &avb;
 	};
 
 	chosen {
-- 
2.7.4

